src/Engine/Scene/Entity.hpp

```diff
@@ -77,6 +81,8 @@ class RA_ENGINE_API Entity : public Core::Utils::IndexedObject
     bool m_isAabbValid {false};
     Core::Aabb m_aabb;
 
+    /// Listeners on Entity Transformation changes
+    Core::Utils::Observable<const Entity*> m_transformationObservers;
```

Comment:
Another option would be to mark `m_transformationObservers` as `mutable`, so the method `transformationObservers` could be kept as `const`. This would simplify user code, don't you think ?

Reply:
I though about that, yes. It could be simpler. I'll test it.